client/components/count/index.jsx

```diff
@@ -1,36 +1,3 @@
-import classnames from 'classnames';
```

Comment:
Do you think we should probably add a comment here explaining the shim cleanup with some links and context as to what is happening here? :)

Reply:
> Do you think we should probably add a comment here explaining the shim cleanup with some links and context as to what is happening here?

I don't mind 👍 

It didn't feel necessary to me since the shim won't exist for a long period of time and git blame will lead them to this PR, but an inline comment only takes a few moments 🙂